tests/virt/cluster/aaq/test_arq.py

```diff
@@ -221,8 +223,16 @@ def test_arq_memory_hotplug(
         hotplugged_resource,
     ):
         vmi_spec_domain = hotplug_vm_for_aaq_test.vmi.instance.spec.domain
+        samples = TimeoutSampler(
+            wait_timeout=TIMEOUT_1SEC, sleep=TIMEOUT_5SEC, func=lambda: application_aware_resource_quota.instance
+        )
+        arq_instance = None
+        for sample in samples:
+            if sample:
+                arq_instance = sample
+                break
```

Comment:
We have a wait() method, you can do `application_aware_resource_quota.wait()`

Reply:
not sure `wait()` is helpful here - he just needs to wait for specific values in already existing resource